### PR TITLE
@ Globals Part 1

### PR DESCRIPTION
**Pull request by rowanseymour**
Adds the models, so we can get mailroom ready

### CODE REVIEW

File: temba/globals/models.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class Global(SmartModel):
+    """
+    A global is a constant value that can be used in templates in flows and messages.
+    """
+
+    MAX_KEY_LEN = 36
+    MAX_NAME_LEN = 36
+    MAX_VALUE_LEN = 640
```
Comment (nicpottier) [inline temba/globals/models.py]: Wonder whether we should allow longer values here than what we typically do for fields / results. Seems unlikely these would end up in either (not a very interesting constant) but maybe shorter prevents JSON lookup tables? Don't feel strongly either way but these aren't expensive to us in any way so maybe making this more like 10k?

File: temba/orgs/models.py [hunk 2]
```diff
@@ -2036,6 +2036,9 @@ def _full_release(self):
         for log in self.http_logs.all():
             log.release()
 
+        for g in self.globals.all():
+            g.release()
```
Comment (nicpottier) [inline temba/orgs/models.py]: ty!

File: temba/globals/models.py (hunk 1 above)
Comment (rowanseymour) [reply temba/globals/models.py]: 10k.. that's like 1 screen of dense text (I can only think in screens).. I think if we do that then the UI should use a textarea.. make it less of a secret that you can do stuff like that?

Comment (nicpottier) [reply temba/globals/models.py]: Other argument here is make the DB max length big but limit the UI to something shorter and see if people request this?

Comment (nicpottier) [reply temba/globals/models.py]: I'm fine with either approach.

Comment (rowanseymour) [reply temba/globals/models.py]: Well it's a `TextField` so it just becomes `text NOT NULL` at a DB level... tho obviously you still end up with a Django migration if you change it. I dunno.. also don't have any strong feelings here.. leaning toward leave as is for now... wait and see how this gets used

**Pull request merged**